src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileOverview/AntigenTestProfileOverviewViewModel.swift

```diff
@@ -0,0 +1,84 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class AntigenTestProfileOverviewViewModel {
+
+	// MARK: - Init
+
+	init(
+		store: AntigenTestProfileStoring,
+		onEntryCellTap: @escaping (AntigenTestProfile) -> Void
+	) {
+		self.store = store
+		self.onEntryCellTap = onEntryCellTap
+		
+		if let existingProfile = store.antigenTestProfile {
```

Comment:
Can we remove the property `antigenTestProfile` at store level?
I can think of a solution to remove the property `antigenTestProfile` and add some logic to the getter of `antigenTestProfiles` which tries to read and add `antigenTestProfile` to the list.